dotnet/src/SemanticKernel.Core/Extensions/KernelSemanticFunctionExtensions.cs

```diff
@@ -175,7 +175,7 @@ public static class KernelSemanticFunctionExtensions
     /// <param name="parentDirectory">Directory containing the plugin directory, e.g. "d:\myAppPlugins"</param>
     /// <param name="pluginDirectoryNames">Name of the directories containing the selected plugins, e.g. "StrategyPlugin"</param>
     /// <returns>A list of all the semantic functions found in the directory, indexed by function name.</returns>
```

Comment:
```suggestion
    /// <returns>A list of all the semantic functions found in the directory, indexed by plugin name.</returns>
```
Isn't this more accurate?